cmd/ris-mirror/rismirror/rismirror.go

```diff
@@ -35,6 +36,38 @@ func (rism *RISMirror) AddTarget(rtrName string, address net.IP, vrfRD uint64, s
 	r.addVRF(vrfRD, sources)
 }
 
+func (rism *RISMirror) RemoveTarget(rtrName string, vrfRD uint64) {
+	rism.routersMu.Lock()
+	defer rism.routersMu.Unlock()
+
+	if _, exists := rism.routers[rtrName]; !exists {
+		log.Errorf("Router %s does not exist, cannot remove cleanly", rtrName)
+		return
+	}
+	r := rism.routers[rtrName].(*Router)
+	if r.GetVRF(vrfRD) == nil {
```

Comment:
Why not make `removeVRF` return an `error` when the VRF does not exist?

Reply:
Sounds good, I've added some more error handling.